reference/v8.json

```diff
@@ -1241,43 +1359,64 @@
   },
   "filter_operator": {
     "type": "enum",
-    "values": [
-      "==",
-      "!=",
-      ">",
-      ">=",
-      "<",
-      "<=",
-      "in",
-      "!in",
-      "all",
-      "any",
-      "none",
-      "has",
-      "!has"
-    ],
+    "values": {
+      "==": {
+          "doc": "The equal-to operator."
```

Comment:
I think the problem with just saying “equal-to operator” is that it isn’t clear that the left-hand side must be the property name and the right-hand side must be a property value. Even saying “left” and “right” could be confusing, though, given that the JSON representation is `["==", "key", "value"]` rather than `["key", "==", "value"]`.